api/src/pcapi/routes/backoffice/finance/finance_incidents_blueprint.py

```diff
@@ -63,6 +63,12 @@ def _get_incidents(
         .outerjoin(finance_models.BookingFinanceIncident.collectiveBooking)
         .outerjoin(educational_models.CollectiveBooking.collectiveStock)
     )
+    print(form.incident_type.data)
+    if form.incident_type.data and len(form.incident_type.data) == 1:
+        if form.incident_type.data[0] == finance_models.IncidentType.OVERPAYMENT.name:
+            query = query.filter(finance_models.FinanceIncident.kind == finance_models.IncidentType.OVERPAYMENT)
+        else:
+            query = query.filter(finance_models.FinanceIncident.kind == finance_models.IncidentType.COMMERCIAL_GESTURE)
```

Comment:
La valeur ayant été validée par wtforms, l'idée est plutôt d'avoir plus simplement : 
```suggestion
        query = query.filter(finance_models.FinanceIncident.kind == finance_models.IncidentType[form.incident_type.data[0]])
```